swatch-contracts/src/main/resources/application.properties

```diff
@@ -285,6 +285,8 @@ quarkus.otel.sdk.disabled=true
 %stage.quarkus.otel.sdk.disabled=false
 %prod.quarkus.otel.sdk.disabled=false
 
+quarkus.hibernate-orm.mapping.timezone.default-storage=normalize-utc
```

Comment:
Is this just preventative or did we see bad behavior without it?  Wondering if our  other quarkus apps have this/need this.